dscan-from-8

```diff
@@ -1,39 +1,42 @@
 DeltaListScanPlan([det], motor, [1, 3, 8])
-stage: (mover: motor), (), {}
 stage: (reader: det), (), {}
-open_run: (None), (), {'plan_type': 'DeltaListScanPlan', 'motors': ['motor'], 'plan_args': {'motor': 'mover: motor', 'steps': '[1, 3, 8]', 'detectors': '[reader: det]'}, 'init_pos': 0, 'detectors': ['det']}
-wait: (None), ('_flyers',), {}
+stage: (mover: motor), (), {}
+open_run: (None), (), {'plan_args': {'steps': [1, 3, 8], 'motor': 'mover: motor', 'detectors': ['reader: det'], 'per_step': 'None'}, 'motors': ['motor'], 'detectors': ['det']}
 checkpoint: (None), (), {}
-set: (mover: motor), (1,), {'block_group': 'A'}
-wait: (None), ('A',), {}
+read: (mover: motor), (), {}
+read: (mover: motor), (), {}
```

Comment:
Yes and yes. This is a side-affect of breaking apart `relative_set` and `reset_positions`. They don't share information.